app/web/src/newhotness/AddViewModal.vue

```diff
@@ -25,7 +25,9 @@
             :value="field.state.value"
             type="text"
             :disabled="wForm.bifrosting.value"
-            @input="(e) => field.handleChange((e.target as HTMLInputElement).value)"
+            @input="
+              (e) => field.handleChange((e.target as HTMLInputElement).value)
+            "
```

Comment:
I've re-run the linter a bunch... my editor does this but `fix-lint` doesn't fix it...